run-local.sh

```diff
@@ -24,6 +24,9 @@ export DB_PASS=calculate-release-dates
 # export HMPPS_AUTH_URL=https://sign-in-dev.hmpps.service.justice.gov.uk/auth
 export HMPPS_AUTH_URL=http://localhost:9090/auth
 
+# export PRISON_API_URL=https://prison-api-dev.prison.service.justice.gov.uk
+export PRISON_API_URL=http://localhost:8080
+
```

Comment:
This seems unrelated to this change, just checking if you meant to include it. (otherwise LGTM)

Reply:
Yes, I'd fallen over it a couple of times and left it in as it makes it easier to setup the local environment if you're not familiar with it.